src/main/webapp/availability.js

```diff
@@ -20,10 +20,14 @@ function onAvailabilityLoad() {
 }
 
 // Toggles a tile from selected (green) to un-selected (white) and vice versa when clicked.
+// Scheduled tiles (red) remain unaffected.
 function toggleTile(tile) {
   let classList = tile.classList;
-  if (classList.contains('table-success') && classList.contains('selected-time-slot')) {
-    classList.remove('table-success', 'selected-time-slot');
+  if (classList.contains('selected-time-slot')) {
+    if (classList.contains('table-success')) {
```

Comment:
Q: how could something be a selected time slot but not be table-success?
Please write about what happens to scheduled slots in the comment of this function.

Reply:
We are saying that both selected and scheduled availabilities are 'selected-time-slot's because that is the class that we use to send all availabilities on the page when the update button is clicked.

Reply:
I think it would be cleaner to have a separate tag to identify scheduled slots. This may already be how you're doing it, but it's hard to tell from this file?